docs/variable-sharing.md

```diff
@@ -10,7 +10,18 @@ Variables are shared by reference for reference types. A consequence of this is
 
 ## Direct data entry with `#!value`
 
+It's common to have text that you'd like to use in a notebook. It might be JSON, CSV, XML, or some other format. It might be in a file, in your clipboard, or on the web. The `#!value` magic command is available to make it as easy as possible to get that text into a variable in your notebook. An important thing to know is that `#!value` is an alias to a  subkernel designed just to hold values. This means that once you store something in it, you can access it from another subkernel using `#!share`.
 
+There are a number of ways to use it. The simplest is to paste some text into the cell. The text will be stored as a string, but unlike using a `string` literal in C#, F#, or PowerShell, there's no need to escape anything.
```

Comment:
Suggestion - Maybe add an ### Example subheading. Just to call this out to the reader

Reply:
Ah right, I still need to add examples for `--from-file` and `--from-url`.